src/error.rs

```diff
@@ -86,6 +86,13 @@ pub enum CmdError {
     /// ["no such window"]: https://www.w3.org/TR/webdriver/#dfn-no-such-window
     NoSuchWindow(wderror::WebDriverError),
 
+    /// The allowed script execution time was exceeded. This variant lifts the
+    /// ["script timeout"] error variant from `Standard` to simplify checking for it in
+    /// user code
+    ///
+    /// ["script timeout"]: https://www.w3.org/TR/webdriver/#dfn-script-timeout-error
+    ScriptTimeout(wderror::WebDriverError),
```

Comment:
Ah, this is pretty unfortunate. Since the public error type here doesn't have `#[non_exhaustive]`, adding a new variant is a breaking change, and we'd have to release a new major version. Is this error variant commonly encountered?

Reply:
Oops! Thanks for bringing that up. The default is 30 seconds, which, for a lot of use cases is plenty. I'm not sure what most people are using the project for but I'd guess this is somewhat common. One example is if you execute an async function and only call the callback on success ("programmer error"), and another would be say waiting for an API call to return. 

I actually have a follow-up commit that has `Client::set_script_timeout` because my use case takes so long.

I'm open to editing however you see fit. Or perhaps a new branch w/ this to anticipate a release some day later.

Reply:
Hmm... I wonder if the right thing to do here is to keep `ScriptTimeout` errors as nested under `Standard`. After all, that's sort of what `Standard` is there for. `NoSuchElement` and `NoSuchWindow` are just explicitly extracted out because they are _so_ common that it'd be annoying to go fishing them out of `Standard` each time.